### PR TITLE
[SKI-27]: Open prometheus port for Oracle to be exposed

### PR DESCRIPTION
**Pull request by Christopher-Li**
### Changelist
- Delete unused flags
- Enable oracle prometheus metrics

### Test Plan
Deployed to dev2

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: protocol/testing/testnet-dev/dev.sh [hunk 1]
```diff
@@ -210,6 +210,10 @@ edit_config() {
 
 	# Disable pex
 	dasel put -t bool -f "$CONFIG_FOLDER"/config.toml '.p2p.pex' -v 'false'
+
+  # Enable Sidecar Prometheus
```
Comment (nivasan1) [inline protocol/testing/testnet-dev/dev.sh:214]: nit: These metrics are being enabled on the application, they are slinky-specific app metrics, update comment accordingly

File: protocol/testing/testnet-local/local.sh [hunk 2]
```diff
@@ -156,12 +156,9 @@ setup_cosmovisor() {
 
 use_slinky() {
   CONFIG_FOLDER=$1
-  # Disable pricefeed-daemon
-  dasel put -t bool -f "$CONFIG_FOLDER"/app.toml 'price-daemon-enabled' -v false
   # Enable slinky daemon
   dasel put -t bool -f "$CONFIG_FOLDER"/app.toml 'oracle.enabled' -v true
 	dasel put -t string -f "$VAL_CONFIG_DIR"/app.toml 'oracle.oracle_address' -v 'slinky0:8080'
-	dasel put -t string -f "$VAL_CONFIG_DIR"/app.toml 'slinky-vote-extension-oracle-enabled' -v 'true'
 }
```
Comment (nivasan1) [inline protocol/testing/testnet-local/local.sh:162]: These are removed as they're set correctly by default?

**Review by nivasan1 — APPROVED** [review]
lgtm w/ nit

File: protocol/testing/testnet-local/local.sh (hunk 2 above)
Comment (Christopher-Li) [reply protocol/testing/testnet-local/local.sh:162]: yes!

**Review by vincentwschau — APPROVED** [review]

**Comment by Christopher-Li** [comment]
https://github.com/Mergifyio backport release/protocol/v5.x

**Pull request merged**